### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by Nick-Tallguy** [hotosm/learnosm#261]
Michael Heißmeier has sent a message to learnosm@hotosm volunteering - am contacting to arrange section & method. Please allocate to me & add Translation label - sorry my access doesn't allow me to do these bits.

**Comment by Nick-Tallguy** [comment]
'in translation' header has now been added to all the files. I've put the already translated iD editor section into the correct folder & have removed duplicates. Should all now be ready for translating.

**Comment by michael63-osm** [comment]
translated 0300-12-31-editing.md and 0300-11-18-editing-techniques.md, so far without creating pull requests;
started translating 0300-11-30-josm-tools.md

**Comment by michael63-osm** [comment]
Created pull requests for 0300-12-31-editing.md, 0300-11-18-editing-techniques.md and 0300-11-30-josm-tools.md which I translated completely, the last file got some new screenshots as well. The final missing bit is a reference to a new samples file which I will change once that file has been added to the repository.

I will continue with 0300-11-29-josm-plugins.md and 0300-11-28-josm-more-tools.md, the latter file will again require a couple of new screenshots with the German interface of josm.

**Comment by althio** [comment]
Thanks @michael63-osm 
> The final missing bit is a reference to a new samples file which I will change once that file has been added to the repository.

Which file or link are we talking about? In which section?
Is someone about to add it to the repository or is it work to do later?

**Comment by michael63-osm** [comment]
@althio, right at the beginning of editing/0300-11-30-josm-tools.md there is a link to tools_menu_sample.osm. This file contains josm menu entries as strings next to the samples used for demonstrating various techniques (you can see that in a number of screenshots later on that page). In order to keep that in sync with the German interface of josm I translated these strings and sent the new version of the file to @Nick-Tallguy together with some screenshots I made. Once he installs the localized sample file I (or someone else) can quickly update the link so that people can download the appropriate version.

By the way, I just see that for some reason the last part of the page is not translated. I thought I did it completely but will surely finish it soon and pull it together with the next translated bucket.

Regards

Michael

**Comment by michael63-osm** [comment]
Created pull requests for 0300-11-29-josm-plugins.md and 0300-11-30-josm-tools.md; I finished the missing parts of the last file and changed the download link to the localized file which Nick installed. Will now work on 0300-11-28-josm-more-tools.md

**Comment by althio** [comment]
Hi @michael63-osm 
> I would welcome suggestions for further translation work.

I think it is also worth reviewing and updating the beginner section.
In particular we are receiving feedback because some pages and screenshots are related to a former interface version of OSM.

I can forward you a recent feedback in German about outdated content in http://learnosm.org/de/beginner/start-osm/#speichere-ein-bild-der-karte

Note: Unfortunately, in English the corresponding content was apparently removed when the page was updated.

**Comment by michael63-osm** [comment]
@althio, it took me a while to figure out what was the original source for that start-osm translation. It turned out to be that file at a previous location (in the root of posts_/en, outside of the beginner folder). The previous section about saving the map was replaced by a section about different styles by @jeffhaack during [this commit](https://github.com/hotosm/learnosm/commit/af738d5d95c347b53c808840bcc6222006429030)
Maybe Jeff can comment a bit about the reasoning for that.

Quite generally, someone who can read German will find that there are subtle differences in the style of the beginners' chapters and the new ones translated by me. I do prefer a somewhat more formal style (not exaggerated though) to rather nerd-like language as it may be appropriate for a presentation at a mapping party, be it only to assure the reader that we take him and his problems serious.

Bottom line: I may well have a look at the beginners chapters and fix issues there, but I would like to rework the pages concerned as well as far as style is concerned. And I still think that showing beginners how they can quickly print a map is useful.

I am still waiting for Nick to install a couple of screenshots for the next editing chapter, would finish that and would then have a closer look at the current status of the German version of the beginners chapter.

**Comment by althio** [comment]
@michael63-osm sorry if my request was unclear. Thank you for tracking down the modification and commit.

About style: Please do as you feel so that your writing is more comfortable and adapted to your readers. I totally support you.

> I still think that showing beginners how they can quickly print a map is useful.

Agreed. IMO It should be added back in English and updated in all languages. No objections @jeffhaack @Nick-Tallguy ?
From user feedback it seems this page and this item are sorely needed. So maybe we could open an issue to address that at least in English?

> waiting for Nick to install a couple of screenshots for the next editing chapter

Sorry if I misunderstand; who is creating the screenshots? You or Nick? If you have the files available we can add them as part of your commits/pull requests, can't we?

Thank you for all your work here!

**Comment by michael63-osm** [comment]
The screenshots are new ones because they show translatable strings from the JOSM GUI. I think that most people who read the German version of learnosm will use the German interface of JOSM so we need different screenshots for those images which do not exclusively show graphics.

The procedure I follow so far is to generate those screenshots, send them to Nick and he installs them under images/de in folders which do not yet exist. I then exchange the respective links in the document and finally pull the documents. As he told me that he currently has limited connectivity this leads to some delay. But I do not consider this a problem - pulling can wait a bit.

**Comment by Nick-Tallguy** [comment]
@michael63-osm should all be up to date now. Connection problems are over - just the jet lag to deal with!  Please let me know if I've missed anything.

**Comment by michael63-osm** [comment]
Created pull requests for a couple of files: the newly translated 0300-11-28-josm-more-tools.md and previous ones where I removed the in-translation header and fixed the titles (they were still in English).

Would now turn to 0200-12-29-start-osm.md from the beginners section. My suggestion is to get the paragraph on printing images up to the current layout, merge the paragraph on different map styles from the English version and polish the whole content somewhat: 
* work a bit on language style as explained previously
* in the 7th bullet item on creating an account I would rather stress that the OSM password should be different from a valuable password like one's mail password, not just say it could be different
* eliminate bullet points 11 and 12; yes, newbies will be reading this but we should not let them feel being treated like computer newbies - I assume a reader will know how to access his email
* the German version still considers Potlatch2 as editor for a first edit - must upgrade this to iD
* add the missing "moving forward" links

**Comment by Nick-Tallguy** [comment]
@michael63-osm I agree with all you are suggesting, especially your points about points 11 & 12. It's one of those lists you tend to skim read & not realise how unnecessary it actually is. Keep up the good work.

**Comment by althio** [comment]
@michael63-osm 
About style: I think most of your ideas will find some resonance in this [Proposed Style Guide for LearnOSM](https://hackpad.com/Proposed-Style-Guide-for-LearnOSM-1JJxS8NIQSX).

@Nick-Tallguy 
I think this is valuable guidance, to be added in a readme/contribute/contributing page. If we agree, we can open an new issue about style and guidance and leave this issue to german translation. ;)

**Comment by Nick-Tallguy** [comment]
@althio I agree

**Comment by michael63-osm** [comment]
Created pull request for 0200-12-29-start-osm.md which is now consistent with the current website

**Comment by michael63-osm** [comment]
Created pull request for 0200-12-24-start-id.md (modified image sent separately to Nick-Tallguy) and started working on 0200-12-18-more-about-josm.md

**Comment by michael63-osm** [comment]
Created pull request for 0200-12-18-more-about-josm.md (modified images sent separately to Nick-Tallguy). Added "move forward" links to all documents in the beginners' chapter in the same way as they appear in the English version.

**Comment by michael63-osm** [comment]
Started converting 0200-12-12-papers.md from Walking Papers to Field Papers

**Comment by Nick-Tallguy** [comment]
@michael63-osm I'm sure you know, but just in case - Field papers have just updated - not sure that anything we have on the site so far is up to date.

**Comment by michael63-osm** [comment]
Created pull request for 0200-12-12-papers.md including modified images. I went through the process on the field papers website making snapshots so this is now more up-to-date than the English version.

**Comment by michael63-osm** [comment]
Start translating 0300-06-15-josm-relations.md

**Comment by michael63-osm** [comment]
Created pull request for 0300-06-15-josm-relations.md.
Next in the queue are the two documents on JOSM presets

**Comment by michael63-osm** [comment]
Created pull request for 0300-08-15-josm-presets.md and 0300-07-15-creating-custom-presets.md together with a small fix in 0300-06-15-josm-relations.md.

**Comment by Nick-Tallguy** [comment]
@michael63-osm I'd like to close this issue now. I've given you collaborator access to the staging site at https://github.com/Nick-Tallguy/Nick-Tallguy.github.io so that you can approve your own pull requests of changes you feel are needed.
I've just implemented the changes on the beginner section & josm sections and would like to thank you for all your help in implementing these.

**Comment by Nick-Tallguy** [comment]
This and other languages are now using the system on Transifex  - closing this. Open to discussion if another system becomes apparent.
